fabric-rendering-v1/src/main/java/net/fabricmc/fabric/api/client/rendering/v1/ArmorRenderer.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.client.rendering.v1;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+import net.minecraft.client.model.Model;
+import net.minecraft.client.render.OverlayTexture;
+import net.minecraft.client.render.RenderLayer;
+import net.minecraft.client.render.VertexConsumer;
+import net.minecraft.client.render.VertexConsumerProvider;
+import net.minecraft.client.render.entity.feature.FeatureRenderer;
+import net.minecraft.client.render.entity.model.BipedEntityModel;
+import net.minecraft.client.render.item.ItemRenderer;
+import net.minecraft.client.util.math.MatrixStack;
+import net.minecraft.entity.EquipmentSlot;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.HashMap;
+import java.util.Objects;
+
+/**
+ * Armor renderers render worn armor items with custom code.
+ * They may be used to render armor with special models or effects.
+ *
+ * <p>The renderers are registered with {@link ArmorRenderingRegistry#register(ArmorRenderer, Item...)}.
+ */
+@FunctionalInterface
+@Environment(EnvType.CLIENT)
+public interface ArmorRenderer {
+	static final HashMap<Item, ArmorRenderer> RENDERERS = new HashMap<>();
```

Comment:
This should probably go in an impl class, exposing the map like that makes it part of the public API. My remark was mostly about having a single API class instead of 2!

Reply:
> Minor things. Make sure you also run `gradlew check` to ensure CI passes when the workflow is approved.

Is there a way to quickly reformat files according to the guidelines or do I have to do that manually?

Reply:
Unfortunately, that has to be done manually.